docs/src/Button.doc.js

```diff
@@ -9,281 +9,287 @@ import PageHeader from './components/PageHeader.js';
+          'Use IconButton if you only need a button with an icon and no text.',
+          'Optional with button-type and submit-type buttons.',
+          'Accessibility: It populates aria-label.  Screen readers read the `accessibilityLabel` prop, if present, instead of `text`.',
+        ]),
+        href: 'accessibilityI',
+      },
       {
         name: 'accessibilityControls',
         type: 'string',
-        description:
-          'Identifies the element whose contents or presence are controlled by the current element. Populates aria-controls.',
-        href: 'accessibility-disclosure',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Specify the `id` of an associated element (or elements) whose contents or visibility are controlled by a button component so that screen reader users can identify the relationship between elements.',
+          'Optional with button-type buttons.',
+          'Accessibility: It populates aria-controls.',
+        ]),
+        href: 'accessibilityII',
       },
       {
         name: 'accessibilityExpanded',
         type: 'boolean',
-        description:
-          'Use this property on elements that can expand to reveal additional information. Populates aria-expanded.',
-        href: 'accessibility-disclosure',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Indicate that a button component hides or exposes collapsible components and expose whether they are currently expanded or collapsed.',
+          'Optional with button-type buttons.',
+          'Accessibility: It populates aria-expanded.',
+        ]),
+        href: 'accessibilityII',
       },
       {
         name: 'accessibilityHaspopup',
         type: 'boolean',
-        description:
-          'Indicates that the element has a popup context menu or sub-level menu. Populates aria-haspopup.',
-        href: 'accessibility-popup',
-      },
-      {
-        name: 'accessibilityLabel',
-        type: 'string',
-        description:
-          'String that clients such as VoiceOver will read to describe the element. Populates aria-label.',
-        href: 'accessibility-popup',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Indicate that a button component controls the appearance of menus and expose whether they are currently opened or closed.',
+          'Optional with button-type buttons.',
+          'Accessibility: It populates aria-haspopup.',
+        ]),
+        href: 'accessibilityII',
       },
       {
         name: 'color',
-        type: `"blue" | "gray" | "red" | "transparent" | "white"`,
+        type: `gray' | 'red' | 'blue' | 'transparent' | 'white'`,
```

Comment:
@AlbertCarreras I think there is a `'` missing before `gray`

Reply:
You think... no... there isnt!